charts/prometheus-elasticsearch-exporter/values.yaml

```
@@ -267,7 +267,8 @@ serviceMonitor:
   enabled: false
   apiVersion: "monitoring.coreos.com/v1"
   #  namespace: monitoring
-  labels: {}
+  labels:
+    release: kube-prometheus-stack
```

This label's value is not applicable to all users, can be included as a comment, though.
```suggestion
labels: {}
  # release: kube-prometheus-stack
```